joblog.go

```diff
@@ -28,10 +32,12 @@ type messageLevel struct {
 	Message string `json:"message"`
 }
 
-func New(url, serviceName string, firstMessage interface{}, options ...func(*JobLog)) (jobLog *JobLog) {
+func New(url, serviceName string, jobName string, actionName string, firstMessage interface{}, options ...func(*JobLog)) (jobLog *JobLog) {
```

Comment:
jobLog = joblog.New(url,
		"test", map[string]interface{}{"log": "this is test 2"}, func(log *joblog.JobLog) {
			log.Disable = true
                        log.JobName ="1111"
                        log.ActionName="2222" 

		})